apps/src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx

```diff
@@ -45,11 +45,21 @@ const WorkshopDashboard = (
         component={SurveyResults}
       />
       <Route
+        path="survey_results(/:workshopId)"
+        breadcrumbs="Workshops"
+        component={SurveyResults}
+      />
```

Comment:
What is this new route? How is it different from the previous `survey_results` routs? Are you sure you want the breadcrumbs to just display `Workshops`?

I think you want a single (combined) route here, with the optional param from line 48 and the breadcrumbs from line 44.